### PR TITLE
Use latest version of boxen, add --width flag

### LINKED ISSUE #73: Upgrade `boxen-cli`
With recent changes.

https://github.com/sindresorhus/boxen-cli

### PR DESCRIPTION
**Pull request by bokub**
- Fixes sindresorhus/boxen#73
- Add `--width` flag
- Custom `--border-style` now accepts 8 characters (old format still working like before)
- Add tests
- Update dependencies
- Switch to `import` syntax

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -32,18 +32,22 @@ const cli = meow(`
 	  │…like everyone│
 	  ╘══════════════╛
 
-	  $ boxen --border-style='1234-|' ASCII ftw!
+	  $ boxen --border-style='1234-|]^' ASCII ftw!
```
Comment (sindresorhus) [inline cli.js]: I think it would be clearer to order it top, bottom, left, right.

Comment (bokub) [reply cli.js]: Fixed!

**Comment by bokub** [comment]
I also updated the `devDependencies` and fixed the new xo errors.

**Comment by sindresorhus** [comment]
Looks good :)

**Comment by bokub** [comment]
Excellent!
Could you please add a `hacktoberfest-accepted` label to this PR so it is counted in my [Hacktoberfest](https://hacktoberfest.digitalocean.com/) objective?

Thank you very much!  :+1:

**Comment by sindresorhus** [comment]
Done

**Pull request merged**